decidim-core/app/packs/src/decidim/index.js

```diff
@@ -105,6 +106,8 @@ const initializer = (element = document) => {
 
   svg4everybody();
 
+  formDatePicker();
```

Comment:
Can we change the `formDatePicker` function to be consistent with the other calls in this same file? Instead of manipulating all the inputs there, we should do the `querySelectorAll` here and receive an input as a parameter: 

```suggestion
  element.querySelectorAll('input[type="datetime-local"], input[type="date"]').forEach((elem) => formDatePicker(elem))
```

(You'd need to adapt `formDatePicker` to this change of course)